### ISSUE TITLE
[Docs] Include information about all of Meshery's integrations

### ISSUE THREAD

**Issue by hamza-mohd** [meshery/meshery#7797]
#### Current State
Meshery's adapters are well represented on the main docs page, but information about the rest of its integrations are missing.

#### Desired State
Include an overview and links to more details about its integrations.

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

**Comment by leecalcote** [comment]
Thanks for raising this issue. With the 200 plus integrations listed on the meshery.io website, you're right: the project needs a description of them in the docs. Would you like to undertake this effort, @jamieplu?

**Comment by jamieplu** [comment]
Yes! Thank you for the very prompt response, @leecalcote. I have a couple of suggestions about how this information can be surfaced. Perhaps, I can start by removing or moving the adapter specific information out of the main overview page. What do you think about this? I know that the end goal is to have MORE details not LESS, but it strikes me that starting there, I might be able to get my feet on the ground and take things from there.

**Comment by leecalcote** [comment]
Good suggestion. The integrations offered by the adapters are perhaps two prominent and misleading in terms of measure his ongoing focus being much more broad across all of cloud native. Yes, I think this is a feasible approach, so long as we follow up with additional details of all of the rest of the integrations. We should probably open up an issue on the other project website, such that that site links each integration to a specific docs page here.

**Comment by jamieplu** [comment]
OK, I'll try to get started very soon. I've worked with Jekyll in the past. I just got a band new GitHub account, so it might not seem that way (I'm starting fresh with this account), but I think I can make a quick impact. Thank you so much for giving me a chance to show what I might be able to do and how I might be able to help the project particularly around the documentation or at least starting there. I've worked with golang for a while now and within Viper and Cobra before - both of which I see that you're using here.
